src/constants/theme/colors.js

```diff
@@ -11,6 +11,7 @@ export const Grays = {
   100: '#50606C',
   60: '#969696',
   30: '#DEDDDA',
+  20: '#EBEFF3',
```

Comment:
Not blocking: we literally made all these colors up so if you ever want to change anything feel free!